src/wkwebkit.cs

```diff
@@ -1169,6 +1191,7 @@ interface IWKPreviewActionItem {}
 
```

Comment:
In two lines above here, a `#if !MONOMAC` is introduced and contains the interface IWKPreviewActionItem and then interface WKPreviewActionItem.

When removing the `#if !MONOMAC` I get the compile error:
`"wkwebkit.cs(1207,34): error CS0527: Type 'NSObject' in interface list is not an interface"`
and I'm not sure what this error means

Reply:
That NSObject is not an interface, it is an object. You need to add an empty interface there.

Reply:
Ah okay that makes more sense of what was going on! It was my own doing haha, it was frustrating me not knowing what was going on here haha